### ISSUE TITLE
Support controlling output delay for CTC loss.

### ISSUE THREAD

**Issue by poor1017** [pytorch/pytorch#43200]
## 🚀 Feature
I need to control output delay for CTC loss.

## Motivation

Original CTC loss impl will cause some delay for output logit, and this delay is uncertain.

## Pitch

Tensorflow builtin ctc has a parameter 'output_delay_', can I get the similar parameter in this project?
